infrastructure/quick-deploy/gcp/multi-stages/gar/parameters.tfvars

```diff
@@ -164,23 +158,5 @@ gar = {
       image = "amazon/aws-efs-csi-driver"
       tag   = "v1.5.1"
     }
```

Comment:
You can delete "aws-efs-csi-driver" too.

Reply:
ok I did it